index.html

```diff
@@ -15,14 +17,19 @@
             <div class="logo">
             <h1>My Logo</h1>
             </div>
-            <a href="#" class="bars-mob"></a>
-            <div class="nav-bar">
-                <ul>
-                    <li><a href="#portfolio-1">portfolio</a></li>
-                    <li><a href="#about-1">about</a></li>
-                    <li><a href="#contact-1">contact</a></li>
+            <img src="icon-bar.svg" class="bars-mob" id="menu-bar" alt="">
+            <!-- <a href="#" class="bars-mob" id="menu-bar"></a> -->
```

Comment:
> Great job implementing the mobile menu, Everything works as expected 🎉

- [OPTIONAL] I noticed that the `menu-icon` doesn't appear, I think the `src` attribute should be 
```
src="images/icon-bar.svg"
```
instead of 
```
src="icon-bar.svg"
```
Kindly update it 💯

- [OPTIONAL] I think it would be better to remove unused code ( Line 21 ) from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)